dist/ember-json-api.js

```diff
@@ -123,24 +149,10 @@ define("json-api-adapter",
           return error;
         }
       },
-      /**
-        Underscores the JSON root keys when serializing.
-
-        @method serializeIntoHash
-        @param {Object} hash
-        @param {subclass of DS.Model} type
-        @param {DS.Model} record
-        @param {Object} options
-        */
-      serializeIntoHash: function(data, type, record, options) {
-        var root = underscore(decamelize(type.typeKey));
-        var snapshot = record._createSnapshot();
-        data[root] = this.serialize(snapshot, options);
-      },
 
       pathForType: function(type) {
         var decamelized = Ember.String.decamelize(type);
-        return Ember.String.pluralize(decamelized);
+        return Ember.String.pluralize(decamelized).replace(/_/g, '-');
```

Comment:
The spec is not specific about `-`, so instead of enforcing it right now, may I suggest adding to the README a piece on how to override `pathForType` to change it from `-` to snakecase.

Personally, I'd leave it as snakecase (because AMS, one of the biggest players in the serializers-land uses snakecase) and inform in the README how to change it to `-`, but that's your choice :smile: